### PR TITLE
Vial Support for Cantor

### PR DESCRIPTION
**Pull request by DrewTChrist**
Hello I'd like to add Vial support for the [Cantor](https://github.com/diepala/cantor). I believe I've set everything up correctly. Please advise if there are any changes that are required.

### CODE REVIEW

**Comment by xyzz** [comment]
It's missing the default keymap, you should add it so that the build succeeds

**Comment by DrewTChrist** [comment]
I have added the default keymap.

**Comment by xyzz** [comment]
Looks like it's still failing to compile

File: keyboards/cantor/rules.mk [hunk 1]
```diff
@@ -0,0 +1,5 @@
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+SPLIT_KEYBOARD = yes
+SERIAL_DRIVER = usart
+VIA_ENABLE      = yes    # Enable VIA
+VIAL_ENABLE      = yes    # Enable VIAL
```
Comment (xyzz) [inline keyboards/cantor/rules.mk]: this should go inside a keymap

**Review by xyzz — CHANGES_REQUESTED** [review]

File: keyboards/cantor/rules.mk (hunk 1 above)
Comment (DrewTChrist) [reply keyboards/cantor/rules.mk]: Alright I moved those lines into keyboards/cantor/vial/rules.mk

**Pull request merged**